src/Get.test.tsx

```diff
@@ -172,6 +229,34 @@ describe("Get", () => {
     });
   });
 
+  describe("with absolute", () => {
```

Comment:
This test is pretty much a duplicate of a test above for composing routes. It is more explicit what it is testing thought. Just pointing it out 😉

Reply:
[Maybe describe the intent in the code as well](https://github.com/contiamo/restful-react/pull/33#discussion_r214679459).